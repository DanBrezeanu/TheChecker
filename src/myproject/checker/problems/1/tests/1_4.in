-2691033
430869809
-407090531
758385476
13146177
628100069
2373106
-758781594
293065936
-314927377
908616923
-413885892
940956541
-258154209
-756567682
78193944
990483731
825190670
334697992
-782439610
244388231
177868440
-933584032
362465157
-155148698
-457082979
707273201
-807614130
-812522901
-479698966
-892002412
-753407388
-794639296
845712568
829025416
856402458
-542213046
-983132436
-462294547
-404080964
-803101196
30205100
990503794
827972072
-75307908
-578399919
961279065
-672804569
817742001
811570939
113691835
256170107
-900217801
579818867
966473250
680911409
-608152557
872034499
-604777093
-52669108
-113954424
-486961631
572240551
851084755
547931153
314326195
-761338111
-129801092
-91452233
-887856057
341782092
167021444
703584574
327286663
18215316
795305726
550907640
-32890728
761143254
-753500778
513394961
335624768
-368820566
-881973326
40678160
-473419606
562517199
312524191
528893716
475135729
-984650170
119802330
702342155
0
