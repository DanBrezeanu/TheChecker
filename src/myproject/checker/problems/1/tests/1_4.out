-984650170
990503794
